src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -52,6 +53,7 @@ case class WorkspaceRequest (
 case class Workspace(
                       namespace: String,
                       name: String,
+                      realm: Option[String] = None,
```

Comment:
you say 
realm: Option[String] = None,

None is the default

instead it should be
realm: Option[String],